### DISCUSSION TITLE
Date field isn't saved

### DISCUSSION THREAD

**Discussion by qbixx** [statamic/cms#6253] [Q&A] [answered]
Hi,

I've created a date field with the settings below. The field then is showing up correct in the Collection that i've assigned the date field to. However when I save the page, the date isn't stored for some reason.

Does anybody perhaps know why this is happening and why the field isn't saved?

![Screenshot 2022-06-27 at 19 21 18](https://user-images.githubusercontent.com/1932778/175999216-1f80df91-6f7a-4942-a406-1526c473e402.png)

**Comment by edalzell** [comment, marked as answer]
Is this in a dated collection?

**Comment by jasonvarga** [comment]
If it's a dated collection then the date will be in the filename, not within the yaml.

**Comment by qbixx** [comment]
That was it! It wasn't a dated collection before. As soon as I changed it the date field was stored correctly. Thanks!
